IBPSA/Utilities/IO/SignalExchange/Read.mo

```diff
@@ -81,5 +99,11 @@ See <a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1059\">#1059</a>.
           rotation=90),
         Line(points={{-16,0},{16,0}},     color={0,0,127},
           origin={70,60},
-          rotation=90)}));
+          rotation=90),
+        Text(
+          extent={{-50,-24},{54,-72}},
+          lineColor={153,153,153},
+          textString=DynamicSelect("", if (Integer(KPIs) == 2 or Integer(KPIs)
```

Comment:
same comment

Reply:
Addressed in https://github.com/ibpsa/modelica-ibpsa/pull/1295/commits/8302e10ff07f1c3cedd10190f951606134372451.